.github/ci/after_make_test.sh

```diff
@@ -6,9 +6,11 @@ set -x
 make install
 
 # Compile examples
+curdir=$PWD
 cd ../examples
 mkdir build;
 cd build;
 cmake ..;
 make;
 ./simple ../simple.sdf;
+cd $curdir
```

Comment:
nit: alternatively we could `pushd` then `popd`

Reply:
Yeah, I considered that, but thought this was simpler because there would have been two `pushd` and `popd`s.

Reply:
I think if you use `pushd` first and `popd` last, then you can use `cd` in the middle

Reply:
Ah, you're right. But I tried it locally and it looks like `pushd` and `popd` are not available in `dash`, the default `sh` binary in Ubuntu. Seeing in https://github.com/ignition-tooling/action-ignition-ci/blob/master/entrypoint.sh#L168 that this script is sourced from a shell script run by `sh` and we don't install `bash` in that docker image, I'm guessing `sh` would still be symlinked to `dash` and not `bash`.

Reply:
ok, `pushd` is too complicated; let's ignore my nit